CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using System.Net.Http;
+using System.Threading.Tasks;
 
-namespace CanHazFunny;
-
-public class JokeService
+namespace CanHazFunny
```

Comment:
```suggestion
namespace CanHazFunny:
```
and remove it's brackets